tests/test_registry.py

```diff
@@ -212,7 +213,11 @@ async def test_find_default(
 
 
 async def test_model_not_ready(model_registry: MultiModelRegistry):
-    slow_model_settings = ModelSettings(name="slow-model", implementation=SlowModel)
+    slow_model_settings = ModelSettings(
+        name="slow-model",
+        implementation=SlowModel,
+        parameters=ModelParameters(version="v1"),
```

Comment:
Similar to above, I think the previous test case was correct, and just started failing because the default model without a version was not present.